sql/custom/world/.gitignore

```diff
@@ -1 +1 @@
-*.sql
+*.sql
```

Comment:
You might have to add a new line to restore the exact content as before, so git will see it as unmodified